modules/update-upgrading-cli.adoc

```diff
@@ -34,49 +34,54 @@ $ oc get clusterversion
 [source,terminal]
 ----
 NAME      VERSION   AVAILABLE   PROGRESSING   SINCE   STATUS
-version   4.8.13     True        False         158m    Cluster version is 4.8.13
+version   4.8.37     True        False         158m    Cluster version is 4.8.37
 ----
 
-. Review the current update channel information and confirm that your channel
-is set to `stable-4.9`:
+. View the available updates and note the version number of the update that
+you want to apply:
 +
 [source,terminal]
 ----
-$ oc get clusterversion -o json|jq ".items[0].spec"
+$ oc adm upgrade
 ----
 +
 .Example output
 [source,terminal]
 ----
-{
-  "channel": "stable-4.9",
-  "clusterID": "990f7ab8-109b-4c95-8480-2bd1deec55ff"
-}
+Cluster version is 4.8.37
+
+Upstream is unset, so the cluster will use an appropriate default.
```

Comment:
Not certain of the history of 'unset'...can this be changed to 'Upstream is not set...' and still be valid contextually?

Reply:
This is `oc` output, so we can't wordsmith here.  But you can file a bug against openshift/oc if you think we should change the wording there.

Reply:
Thanks for the clarification!!